swap_meet/clothing.py

```diff
@@ -1,2 +1,30 @@
-class Clothing:
-    pass
\ No newline at end of file
+from .item import Item
+
+class Clothing(Item):
+    '''
+    Class that represents a clothing item.
+    
+    ...
+
+    Attributes
+    - - - - - -
+    category : str
+    condition : int
+
+    Methods
+    -------
+    str()
+    condition.description()
+    '''
+
+    def __init__(self, condition = 0.0):
+        '''
+        Constructs attributes for Clothing object from parent class; default values: category = "Clothing", condition = 0.0.
+        '''
+        super().__init__(category = "Clothing", condition = condition)
```

Comment:
👍  great job hardcoding the `category` param inside the `super`. That way a user can't accidentally or intentionally change the attribute!